lib/internationalization/ISO_handler.fz

```diff
@@ -0,0 +1,2 @@
+public ISO_handler : int_handler is 
```

Comment:
please add the header to every file. 

also, add a small comment explaining what this file does.